hbase-server/src/main/java/org/apache/hadoop/hbase/master/normalizer/RegionNormalizerWorker.java

```diff
@@ -181,10 +189,19 @@ public void run() {
     final TableDescriptor tblDesc;
     try {
       tblDesc = masterServices.getTableDescriptors().get(tableName);
-      if (tblDesc != null && !tblDesc.isNormalizationEnabled()) {
-        LOG.debug("Skipping table {} because normalization is disabled in its table properties.",
-          tableName);
-        return Collections.emptyList();
+      boolean normalizationEnabled;
+      if (tblDesc != null) {
+        String defined = tblDesc.getValue(TableDescriptorBuilder.NORMALIZATION_ENABLED);
+        if(defined != null) {
+          normalizationEnabled = tblDesc.isNormalizationEnabled();
+        } else {
+          normalizationEnabled = this.defaultNormalizerValue;
+        }
+        if (!normalizationEnabled) {
+          LOG.debug("Skipping table {} because normalization is disabled in its table properties.",
+            tableName);
```

Comment:
Let's make it
```
        if (!normalizationEnabled) {
          LOG.debug("Skipping table {} because normalization is disabled in its table properties and normalization is also disabled at table level by default (config: {}).",
            tableName, NORMALIZATION_ENABLED_CONF_KEY);
          return Collections.emptyList();
        }
```